v2/launcher_container/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:alpine
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+# Copy the code into the container
+COPY cmd /build/cmd
+COPY component /build/component
+COPY metadata /build/metadata
+COPY third_party /build/third_party
+COPY go.mod /build/.
+COPY go.sum /build/.
```

Comment:
I wonder why we need to copy them one by one.
Can we do `COPY . /build`?

Reply:
Done.